docs/src/modules/utils/useQueryParameterState.ts

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import debounce from 'lodash/debounce';
```

Comment:
We know have our own `debounce` util from '@mui/utils', maybe we should use it here?